04/app.js

```diff
@@ -0,0 +1,48 @@
+function getNumbers(min, max) {
+  const arr = [];
+  const lengthArr = 20;
+  for (let i = 0; i < lengthArr; i++) {
+    arr.push(createRandomNumbers(min, max))
+  }
+  return arr;
+}
+
+function createRandomNumbers(min, max) {
+  return Math.round(Math.random() * (max - min) + min)
+}
+
+const result = getNumbers(1, 30);
+console.log(result);
+
+
+function sortNumbers(GetNumbersResult) {
+  const leftNumbers10 = GetNumbersResult.sort(function (a, b) {
+    return a - b;
+  }).slice(-10); // dlaczego podajac (0,10) zwraca mi 10 elementow a nie 9? nie powinno byc, że od indexu 0 do 10 ale bez niego? jesli chciałbym 10 elementow to powinienem podać (0,11)
```

Comment:
Cytat z MDN (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice): `start to end (end not included) where start and end represent the index of items in that array. `